webcompat/helpers.py

```diff
@@ -455,6 +455,12 @@ def api_request(method, path, params=None, data=None):
     resource = request_method(method, path, headers=request_headers,
                               params=params, data=data)
     if resource.status_code != 404:
+        try:
+            if 'pull_request' in json.loads(resource.content):
+                return ('Pull Requests are Forbidden.', 403,
+                         get_response_headers(resource))
```

Comment:
Hm, I think we're sending back a JSON mimetype at this point, but the content is not valid JSON. To fix, even though nothing in this PR relies on it.

Reply:
Updated.